Meadowlark-js/services/meadowlark-aws-lambda/.env.example

```diff
@@ -29,5 +29,12 @@ ES_PASSWORD=admin
 
 
 #### Configurable backend plugin PoC - set to an npm package name
-#BACKEND_PLUGIN_NAME=@edfi/meadowlark-aws-dynamodb-opensearch
-BACKEND_PLUGIN_NAME=@edfi/meadowlark-local-mongodb-opensearch
+#DOCUMENT_STORE_PLUGIN=@edfi/meadowlark-aws-dynamodb-opensearch
+DOCUMENT_STORE_PLUGIN=@edfi/meadowlark-mongodb-backend
+
+#QUERY_HANDLER_PLUGIN=@edfi/meadowlark-aws-dynamodb-opensearch
+QUERY_HANDLER_PLUGIN=@edfi/meadowlark-opensearch-backend
+
+#LISTENER1_PLUGIN=@edfi/meadowlark-opensearch-backend
+
+#LISTENER2_PLUGIN=@edfi/meadowlark-test-listener
```

Comment:
Would it make sense to move this to the `Meadowlark-js` directory?

Reply:
Well, I see that this wouldn't make sense. However, I find it very odd to have to dig this deep into the directory structure just to start things up.